.circleci/config.yml

```diff
@@ -1663,6 +1783,14 @@ workflows:
             - slack-secrets
       - integration-tests-all
 
+  daily-maestro-e2e-tests:
+    when:
+      and:
+        - equal: [scheduled_pipeline, << pipeline.trigger_source >>]
+        - equal: ["maestro_e2e_tests", << pipeline.schedule.name >>]
+    jobs:
+      - <<: *maestro-e2e-test-job
```

Comment:
I'm curious. What's this syntax to run the job? Does it mean running the job with all the matrix combinations?

Reply:
Yes so it should run daily, as well as when all tests are triggered. And indeed it will run all matrix combinations (just the 4 backend environments)